src/components/AnimatedButton/AnimatedButton.tsx

```diff
@@ -76,6 +77,8 @@ const AnimatedButton: React.FC<Props> = ({
             progress === 'failed'
             ? classes.buttonRelease
             : undefined,
+          content === 'Connect a wallet' && classes.buttonSelectDisabled,
```

Comment:
instead of hardcoding special cases use className prop where you're using that component and set styles through this